rcon/extended_commands.py

```diff
@@ -586,27 +528,26 @@ def get_gamestate(self) -> GameState:
         }
 
     @ttl_cache(ttl=2, cache_falsy=False)
-    def team_sizes(self) -> Tuple[int, int]:
+    def get_team_sizes(self) -> Tuple[int, int]:
```

Comment:
by changing the name to get_ you're exposing those to the API layer too

Reply:
Whoops, that was intentional but in retrospect I should not I think since we already have that information available elsewhere and in a nicer format.